build_sysext

```diff
@@ -134,11 +143,34 @@ trap cleanup EXIT
 ARCH=$(_get_sysext_arch "${FLAGS_board}")
 cleanup
 
+# If we need to handle pkginfo squashfs files, create mount points under
+# ${BUILD_DIR}/base-pkginfo, mount the squashfs images, and add the mount paths to
+# the list of lowerdirs.
+pkginfo_lowerdirs=""
+if [[ -n "${FLAGS_base_pkginfo}" ]] ; then
+  for entry in $(echo ${FLAGS_base_pkginfo} | sed 's/:/ /g'); do
+    ppath="$(readlink -f "${entry}")"
+    if [[ ! -f "${ppath}" ]] ; then
+      error "--base_pkginfo contains invalid entries."
+      error "Pkginfo file '${ppath}' does not exist."
+      die "Full --base_pkginfo: '${FLAGS_base_pkginfo}'"
+    fi
+
+    pfile="$(basename "${ppath}")"
+    pmdir="${BUILD_DIR}/base-pkginfo/${pfile}"
+    mkdir -p "${pmdir}"
```

Comment:
Should we check if `pmdir` didn't exist before to avoid silently mounting the squashfs over already existing mount?

Reply:
I think this is fine - it's up to the caller to pass an appropriate `BUILD_DIR` anyway.

Reply:
I was thinking not in terms of BUILD_DIR, but rather in terms of possible conflicts where basename for two different ppaths will return the same value, thus reusing the same directory for the second ppath.